packages/qa/pages/StudyUpdatePage.ts

```diff
@@ -95,6 +96,7 @@ export default class StudyUpdatePage {
     this.requestSupport = page.locator('[data-testid="request-support"]')
     this.updateSuccessBanner = page.locator('.govuk-notification-banner.govuk-notification-banner--success')
     this.updateSuccessContent = page.locator('.govuk-notification-banner__heading')
+    this.updateValidationBanner = page.locator('')
```

Comment:
Minor and appreciate it might be updated in another PR:

- is the locator meant to take in an empty string?